src/Employer/Employer.Web/Startup.Configure.cs

```diff
@@ -43,17 +44,50 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, IOptions
 
                 app.UseHsts(hsts => hsts.MaxAge(365));
             }
-            
+
+            if (env.IsDevelopment())
+            {
+                IdentityModelEventSource.ShowPII = true;
+            }
             // Add Content Security Policy
+            app.Use(async (context, next) =>
+            {
+                context.Response.Headers["Content-Security-Policy"] =
+                    "default-src 'self' 'unsafe-inline' https://*.zdassets.com https://*.zendesk.com wss://*.zendesk.com wss://*.zopim.com; " +
+                    "img-src 'self' *.google-analytics.com https://*.zdassets.com https://*.zendesk.com wss://*.zendesk.com wss://*.zopim.com; " +
+                    "script-src 'self' 'unsafe-inline' " +
+                    "*.googletagmanager.com *.postcodeanywhere.co.uk *.google-analytics.com *.googleapis.com https://*.zdassets.com https://*.zendesk.com wss://*.zendesk.com wss://*.zopim.com; " +
+                    "font-src 'self' data:;";
+                await next();
+            });
```

Comment:
Do we need this middleware to add the header if we are already have the UseCsp middleware below setting all the same values?